packages/pluginsdk/scripts/scaffold.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+const path = require('path');
+const fs = require('fs');
+const child_process = require('child_process');
+const readlineSync = require('readline-sync');
+
+const pluginNameQuestion = 'Enter the short name for your plugin (default: myplugin): ';
```

Comment:
Very basic choices for now.  This choice changes the value of package.json and also the default output directory.
At some point, we will want to add command line arg parsing (`yargs` for nodejs) for use with tooling automation.